eng/packaging.targets

```diff
@@ -202,6 +202,18 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="WarnOnProjectReferenceToFrameworkAssemblies"
+          BeforeTargets="IncludeTransitiveProjectReferences"
```

Comment:
For my own education: How did you know, or why did you decide, to get this target called before `IncludeTransitiveProjectReferences` specifically?